Dockerfile

```diff
@@ -67,6 +72,7 @@ RUN apt-get update && apt-get install -y \
 	xfsprogs \
 	libzfs-dev \
 	tar \
+	linux-image-`uname -r` \
```

Comment:
That note probably needs updated as `contrib/` definitely has the usual sense of "contributed content" that may not be developed alongside the main body of code, but the Dockerfiles I pointed happen to also live under that sub-tree and are used to create the official build packages that are used by the official install scripts, so they are definitely maintained appropriately. I'll ping @jfrazelle in case she wants to add any info about future ideas to keep all these Dockerfiles in sync as changes/features show up in the main development Dockerfile.  No pressure @jfrazelle if there is nothing to share :) Just wanted to give you an opportunity to add anything of value about the process.